todo.js

```diff
@@ -1,104 +1,48 @@
-
-//click add to create items and append to list
+function newTaskOnKeyPress(){
+    newTaskOnClick;
+}
 
 function newTaskOnClick (){
-
-	//create item, delete button and checkbox
-
-	var todoList = document.getElementById('list');
+    
+//click add to create items and append to list
 
 	var todoItem = document.createElement('li');   
 	var deleteButton = document.createElement('span');
-	var checkBox = document.createElement('input'); 
+	var checkBox = document.createElement('input');
 
 	checkBox.className = "check";
 	checkBox.type = "checkbox";
 	deleteButton.innerText = "x";
 	deleteButton.className = "Remove";
-
-
-	//append and insert text node, delete button  and check box
-
+	 
 	var inputValue = document.getElementById('input').value;
 	var txt = document.createTextNode(inputValue);
 	todoItem.appendChild(txt);
 	todoItem.appendChild(deleteButton);
 	todoItem.insertBefore(checkBox, txt);
 	 
-
-	// if input is blank the if statement will ask user to input task
-
-		if (inputValue === ''){
-			alert ('please enter a task');
-		}
-		else {
-			todoList.appendChild(todoItem);
-
-
-			//give unique id to each list item.
-
-			var grabList = document.querySelectorAll('li');
-			for (var i = 0; i < grabList.length; i++){
-				var grabLi = grabList[i];
-			}
-			grabLi.id = 'item' + i;
-		}
-		document.getElementById('input').value;
-		storeItems();
-	 
-		
+	if (inputValue === ''){
+		alert ('please enter a task');
+	}
+	else {
+		document.getElementById('list').appendChild(todoItem);
+	}
+    document.getElementById('input').value;
```

Comment:
fix indentation here. line 30 does not align well with  the if else statement above